src/main/kotlin/com/sourcegraph/cody/sidebar/VSCodeInjected.kt

```diff
@@ -0,0 +1,40 @@
+package com.sourcegraph.cody.sidebar
```

Comment:
Could we put this as real script in `/resources` and load using class loader when needed?